cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -226,73 +222,76 @@ void WholeStageResultIterator::collectMetrics() {
       }
       // Special handing for Filter over Project case. Filter metrics are
       // omitted.
-      numOfStats += 1;
+      statsNum += 1;
       continue;
     }
-    numOfStats += planStats.at(nodeId).operatorStats.size();
+    statsNum += planStats.at(nodeId).operatorStats.size();
   }
 
-  metrics_ = std::make_shared<Metrics>(numOfStats);
-  int metricsIdx = 0;
+  metrics_ = std::make_unique<Metrics>(statsNum);
+
+  int i = 0; // stands for Metric Index
```

Comment:
Why not name it as `metricIndex ` and remove this comment?

Reply:
OK